lib/widgets/movie_detail_play_button.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import '../constants/measures_constants.dart';
+import '../constants/string_constants.dart';
+
+class MovieDetailPlayButton extends StatelessWidget {
+  const MovieDetailPlayButton({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(
+        left: MeasuresConstants.playButtonLeftRightPadding,
+        right: MeasuresConstants.playButtonLeftRightPadding,
+      ),
```

Comment:
You can change this from EdgeInsets.only to EdgeInsets.symmetric and change left/right with horizontal

Reply:
Changed